### PR TITLE
Installation instructions: explain how to install `pipx`

### PR DESCRIPTION
**Pull request by ilyagr**
https://github.com/python-poetry/poetry/pull/8737#issuecomment-1891190411 brought this to my mind.

**Draft:** I haven't previewed the result yet. Apparently, somebody needs to tag this with the appropriate label before Github generates a preview automatically.

### CODE REVIEW

File: docs/_index.md [hunk 1]
```diff
@@ -45,6 +45,14 @@ See the **advanced** installation instructions to use a preview or alternate ver
 
 {{< steps >}}
 {{< step >}}
+**Install `pipx`**
```
Comment (radoering) [inline docs/_index.md:48]:
```suggestion
**Install pipx**
```

File: docs/_index.md [hunk 2]
```diff
@@ -45,6 +45,14 @@ See the **advanced** installation instructions to use a preview or alternate ver
...
+
+If `pipx` is not already installed, you can follow any of the options in the
+[official `pipx` installation instructions](https://pipx.pypa.io/stable/installation/).
```
Comment (radoering) [inline docs/_index.md:51]:
```suggestion
[official pipx installation instructions](https://pipx.pypa.io/stable/installation/).
```

**Review by radoering — COMMENTED** [review]
I'd omit the backticks in the heading and the link (see preview).

**Comment by ilyagr** [comment]
Sure, done.

**Review by radoering — APPROVED** [review]

**Pull request merged**